transport/src/main/java/io/netty5/channel/AbstractChannel.java

```diff
@@ -71,6 +72,10 @@ public abstract class AbstractChannel extends DefaultAttributeMap implements Cha
     /** true if the channel has never been registered, false otherwise */
     private boolean neverRegistered = true;
 
+    private static final AtomicIntegerFieldUpdater<AbstractChannel> WRITABLE_UPDATER =
+            AtomicIntegerFieldUpdater.newUpdater(AbstractChannel.class, "writable");
+    private volatile int writable = 1;
```

Comment:
I did preserve the old behaviour of have a Channel writable from the start. That said we might want to reconsider this in another PR and only mark it writable once the connection is established.